.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
IDE ignore should be part of your local global gitignore, otherwise we will get more IDE ignores than real app-specific ignores as there are simply far too much IDE files and folders out there.